config-schema.json

```
@@ -0,0 +1,344 @@
+{
+	"type": "object",
+	"properties": {
+		"server": {
+			"type": "object",
+			"properties": {
+				"host": {
+					"type": "string"
+				},
+				"port": {
+					"type": "integer"
+				},
+				"apiKey": {
+					"type": "string",
+					"minLength": 16,
+					"maxLength": 16

```

```suggestion
					"minLength": 32,

```
The current default is 16 bytes (32 hex characters). Since we support custom lengths with the standard being 16 bytes, this should probably be changed.

```
Usage:
  omegabrr [command] [flags]

Commands:
  generate-token Generate an API Token (optionally call with --length <number>)

Flags:
  --length <number>    Length of the generated API token (default 16)
```